tasks/main.yml

```diff
@@ -28,8 +37,54 @@
     user_install: false
   with_items: "{{ fluentd_plugins }}"
 
+- name: retrieve initial ExecStart
+  shell: 'grep -i "ExecStart=.*/.*" /lib/systemd/system/multi-user.target.wants/td-agent.service | sed -r "s/^([^\/])*(\/.*)/\2/"'
+  register: initial_cmd
+  tags:
+  - never
+  - aggregator
+
+- name: Ensure unit directory is present
+  file:
+    path: /etc/systemd/system/td-agent.service.d
+    state: directory
+    mode: '0755'
+  tags:
+  - never
+  - aggregator
+
+- name: Ensure override file is absent
+  file:
+    path: /etc/systemd/system/td-agent.service.d/override.conf
+    state: absent
+  tags:
+  - never
+  - aggregator
```

Comment:
Is it mandatory to ensure if the file is absent?

Reply:
Is there a better way to clear the file with ansible?

Reply:
Yes, using a template file :)

Reply:
Then you can remove merge `reset ExecStart` task with `Add vault secret`

Reply:
Much better, thanks